assets.go

```diff
@@ -2,4 +2,8 @@
 
 package main
 
+import "net/http"
+
 const production = false
```

Comment:
Should it really be called _production_ when the build tag itself is called _dev_ ?